OsmAnd/src/net/osmand/plus/OsmandApplication.java

```diff
@@ -894,8 +894,14 @@ public AssetManager getAssets() {
 
 	@Override
 	public Resources getResources() {
-		Resources localizedResources = localeHelper.getLocalizedResources();
-		return localizedResources != null ? localizedResources : super.getResources();
+		OsmandMap map = getOsmandMap();
```

Comment:
I wrote this part, it doesn't look very nice as it access activity from application, though it looks it much simpler approach however it's bad to use resources from application but it requires refactoring for the whole application